graph-server/src/renderer/index.js

```diff
@@ -0,0 +1,69 @@
+const { processSvg } = require('./image-util');
+const BUNDLES = require('../assets');
+
+
+const getSvgImpl = {
+    puppeteer: require('./impl/puppeteer'),
+    phantomjs: require('./impl/phantomjs')
+};
+const DEFAULT_IMPLEMENTATION = 'puppeteer';
+
+
+const SERVICE_ROOT = 'https://waterservices.usgs.gov/nwis';
+const PAST_SERVICE_ROOT = 'https://nwis.waterservices.usgs.gov/nwis';
```

Comment:
Eventually this should come from config. We could potentially use local urls for this?

Reply:
Agree... on the watermark/legend branch I started grabbing them from environment variables. Don't know if that is best, or a config file would be better.